in-toto-spec.md

```diff
@@ -479,7 +479,7 @@ All signed files (i.e., link and layout files) have the format:
 }
 ```
 
-Where, ROLE is a dictionary whose "\_type" field describes the metadata type (as
+Where, ROLE is a dictionary whose `"\_type"` field describes the metadata type (as
```

Comment:
Now that we mark this up as verbatim text, the escape character `\` becomes visible, so we should remove it.

Reply:
Oops, you are right. I didn't see this. I will make a PR ready that removes the backslash